content/docs/deployment/clients/_index.md

```diff
@@ -334,65 +690,62 @@ Now right click the GPO object and select "Edit":
 
 ![Edit GPO](5.png)
 
-We will create a new scheduled task. Rather than schedule it at a
-particular time, we will select to run it immediately. This will force
-the command to run as soon as the endpoint updates its Group Policy
-settings, because we don't want to wait for the next reboot of the
-endpoint.
+We will create a new scheduled task. Rather than schedule it at a particular
+time, we will select to run it immediately. This will force the command to run
+as soon as the endpoint updates its Group Policy settings, because we don't want
+to wait for the next reboot of the endpoint.
 
 ![Scheduled Task](6.png)
 
-Next we give the task a name and a description. In order to allow
-Velociraptor to access raw devices (e.g. to collect memory or NTFS
-artifacts) we can specify that the client will run at
-`NT_AUTHORITY\SYSTEM` privileges and run without any user being logged
-on.
+Next we give the task a name and a description. In order to allow Velociraptor
+to access raw devices (e.g. to collect memory or NTFS artifacts) we can specify
+that the client will run at `NT_AUTHORITY\SYSTEM` privileges and run without any
+user being logged on.
 
-It's also worth ticking the "hidden" checkbox here to prevent a
-console box from appearing.
+It's also worth ticking the "hidden" checkbox here to prevent a console box from
+appearing.
 
 ![Adding a new task](7.png)
 
-Next click the Actions tab and add a new action. This is where we
-launch the Velociraptor client. The program will simply be launched
-from the share (i.e. `\\DC\Deployment\velociraptor.exe`) and we give
-it the arguments allowing it to read the provided configuration file
+Next click the Actions tab and add a new action. This is where we launch the
+Velociraptor client. The program will simply be launched from the share (i.e.
+`\\DC\Deployment\velociraptor.exe`) and we give it the arguments allowing it to
+read the provided configuration file
 (i.e. `--config \\DC\Deployment\client.config.yaml client -v`).
 
```

Comment:
Is it worth mentioning that this process can be used to also install the msi by just running msiexec /i or using assigned MSI instead of scheduled task?

Reply:
Yes, good idea. I've added it to the section that describes the MSI install. Also updated the Microsoft link.